File: Programación y Algoritmos/Actividad 15/ventas.py
contador = 0
total = 0
venta = 1
destacadas = 0

while venta > 0:
    venta = float(input("Introduzca el monto de la venta (0 para terminar): "))
    if venta > 0:
        contador += 1
        total += venta
        if venta >= 1000:
            destacadas += 1

print(f"Ventas totales: {contador}")
print(f"Monto acumulado: ${total}", end="")
if total > 5000:
    print("  ¡Excelente jornada de ventas!", end="")
print("")
print(f"Promedio de la venta: ${total/contador}")
print(f"Ventas destacadas: {destacadas}")

File: Programación y Algoritmos/Actividad 15/ventas.cpp
#include <iostream>
#include <string.h>
using namespace std;

float contador = 0;
float destacadas = 0;
float total = 0;
float venta = 1;
int main()
{

    while (venta > 0)
    {
        cout << "Introduzca el monto de la venta (0 para terminar): ";
        cin >> venta;
        if ( venta > 0  )
        {
            contador++;
            destacadas = (venta > 1000) ? destacadas+1: destacadas;
            total += venta;
        }
        

    }
    string mensaje = "";
    if (total > 5000)
    {
        mensaje = "Excelente jornada de ventas!";
    }
    cout << endl;
    cout << "Ventas totales: " << contador << endl;
    cout << "Monto acumulado: $" << total << "  " << mensaje << endl; 
    cout << "Promedio de venta: $" << total/contador << endl;
    cout << "Ventas destacadas: " << destacadas << endl;

}